admin-dev/themes/new-theme/js/components/components-map.ts

```diff
@@ -96,4 +96,9 @@ export default {
     selector: '.autoload_rte',
     selectorClass: 'autoload_rte',
   },
+  contextualNotification: {
+    close: '.contextual-notification .close',
+    headerMultiShop: '.header-multishop',
```

Comment:
This one should actually be in `ComponentsMap.multistoreHeader` it seems more fitting that the multistore header defines this selector even if it's used by itself but for other components that depend on it

It does create a dependency with the multistore header component, but it's indeed the case so it makes that components map are dependent with each other if the components are as well

Reply:
:+1: